plugin/kotel/version.go

```diff
@@ -0,0 +1,21 @@
+package kotel
+
+import "runtime/debug"
+
+// Version is the current release version of the kotel instrumentation.
+func Version() string {
+	info, ok := debug.ReadBuildInfo()
+	if ok {
+		for _, dep := range info.Deps {
+			if dep.Path == "github.com/twmb/franz-go/plugin/kotel" {
+				return dep.Version
+			}
+		}
+	}
+	return "unknown"
+}
+
+// SemVersion is the semantic version to be supplied to tracer/meter creation.
+func SemVersion() string {
+	return "semver:" + Version()
```

Comment:
Is this `semver:` prefix standard for otel? I'm wondering why SemVersion is needed / exists, vs. just using Version.

Reply:
We are following the same versioning pattern used in other OpenTelemetry libraries as can be seen in the link provided [1].

[1] https://github.com/search?q=repo%3Aopen-telemetry%2Fopentelemetry-go-contrib%20semversion&type=code